src/SpecFlow.NetCore/Fixer.cs

```diff
@@ -43,14 +44,27 @@ public Fixer()
 		public void Fix(DirectoryInfo directory)
 		{
 			WriteLine("Current directory: " + directory);
-			var xproj = GetXproj(directory);
+		    featureFiles = directory.GetFiles("*.feature", SearchOption.AllDirectories);
+		    var generatedFileExists = featureFiles.Where(f => !File.Exists(f.FullName + ".cs")).ToList();
+
+            var xproj = GetXproj(directory);
```

Comment:
Happy to adapt these diffs look terrible :( I changed my vs tab settings but would like to keep settings for work projects and such. Perhaps https://visualstudiogallery.msdn.microsoft.com/c8bccfe2-650c-4b42-bc5c-845e21f96328 is an acceptable solution? 

We'll see if tab settings will fix this issue next PR: